map_frame_id: 'map'
robot_frame_id: 'base_link'
map_service_name: '/dynamic_map'
goal_topic: '/move_base_simple/goal'
map_topic: '/map'
occ_map_topic: "/occ_map"
map_metadata_topic: '/map_metadata'
path_topic: '/global_path'
inflation_radius: 0.2
request_position_x: 0.0
request_position_y: 0.0
request_length_x: 100.0
request_length_y: 100.0
position_error_endurance: 0.05
global_planner_time_inverval: 0.05
occ_threshold: 25

RRT_Star/steer_length: 0.1
RRT_Star/search_radius: 1.0
RRT_Star/search_time: 0.03
RRT_Star/max_tree_node_nums: 100000
